deployments/helm-chart/templates/controller-servicemonitor.yaml

```diff
@@ -0,0 +1,15 @@
+{{- if .Values.controller.serviceMonitor.create }}
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: {{ .Release.Name }}
```

Comment:
I think it would make sense to make the name customizable and maybe define it in `_helpers.tpl` like we do for other names

Reply:
Excellent callout, thanks! I've added that in.